<launch>

    <include file="$(find omnicore_launcher)/launch/real_robot.launch">
        <arg name="robot" value="gofa"/>
    </include>

    <include file="$(find schunk_hardware_interface)/launch/gripper.launch">
    </include> 
    
    <arg name="manual_move_confirm" default="true"/>

    <node pkg="gofa_schunk_services" type="gofa_schunk_services" name="gofa_schunk_services_node" output="screen">
        <param name="manual_move_confirm" value="$(arg manual_move_confirm)" />
    </node>

</launch>